topics/computational-chemistry/tutorials/covid19-docking/tutorial.md

```diff
@@ -46,18 +44,17 @@ tutorial. -->
+Early in March 2020, the Diamond Light Source completed a successful fragment screen on MPro, which provided 55 fragment hits (see their [press release](https://www.diamond.ac.uk/covid-19/for-scientists/Main-protease-structure-and-XChem.html) here). In an effort to identify candidate molecules for binding, InformaticsMatters, the XChem group and the European Galaxy team joined forces to construct and execute a Galaxy workflow for performing and evaluating molecular docking on a massive scale.
 
 An initial list of 41,000 candidate molecules was assembled by using the Fragalysis fragment network to elaborate from the initial fragment hits, as described [here](https://diamondlightsource.atlassian.net/wiki/spaces/FRAG/pages/8323192/The+Astex+Fragment+network). These were used as inputs for the docking and scoring workflow. The workflow consists of the following steps, each of which was carried out using tools installed on the European Galaxy server:
 1. Charge enumeration of the 41,000 candidate molecules selected based on the fragment hits.
 2. Generation of 3D conformations based on SMILES strings of the candidate molecules.
 3. Docking of molecules into each of the MPro structures using rDock.
-4. Evaluation of the docking poses using a deep learning approach [3] developed by the XChem group and collaborators.
-
-The original study required almost 20 years of CPU time, not counting GPU resources consumed. This is obviously not reproducible as a tutorial. Therefore, we will repeat the workflow with a small library of just 100 molecules, on a single MPro fragment structure. Links will be provided to all the original Galaxy histories, with notes to explain where and why things were done differently to the tutorial.
+4. Evaluation of the docking poses using a TransFS, a deep learning approach ({% cite transfs %}) developed by the XChem group and collaborators, and SuCOS scoring ({% cite sucos %}), which compares the poses with the structures of the original fragment hits.
 
- <!-- employing augmentation of training data with incorrectly docked ligands to prompt the model to learn from protein-ligand interactions. The algorithm was deployed on the European Galaxy server inside a Docker container, thanks to work by InformaticsMatters and the European Galaxy team. -->
+The original study required almost 20 years of CPU time, not counting GPU resources consumed. This is obviously not reproducible as a tutorial. Therefore, we will repeat the workflow with a small library of just 100 molecules, on a single MPro fragment structure. Links will be provided to original Galaxy histories, with notes to explain where and why things were done differently to the tutorial.
 
+![MPro structure, with a fragment bound]({% link topics/computational-chemistry/images/mpro.png %} "Structure of MPro, with a fragment bound. View in NGL ({% cite ngl %}) [here](https://usegalaxy.eu/u/sbray/v/mpro-x0072).")
```

Comment:
```
[here](https://usegalaxy.eu/u/sbray/v/mpro-x0072)
```
This link (in a image caption) does not work. Not sure if this is fixable.

Reply:
have you tried HTML?

Reply:
yes, I tried HTML. Not sure what else to try. Otherwise I will just link in the main text.

Reply:
Tried again and got it to work